docs/quickstart_and_tutorials/development/getting_started.rst

```diff
@@ -0,0 +1,49 @@
+.. _getting_started:
```

Comment:
A stab at https://github.com/astropy/halotools/issues/932

Reply:
Wow, thanks for this.